packages/dialect-react-ui/components/DisplayAddress/index.tsx

```diff
@@ -1,25 +1,24 @@
 import type { PublicKey } from '@solana/web3.js';
-import { useIdentity } from '@dialectlabs/react-sdk';
+import { Identity, useIdentity } from '@dialectlabs/react-sdk';
 import { TwitterIcon } from '../Icon/Twitter';
 
 import { A } from '../common/preflighted';
 import { Loader } from '../common';
+import { shortenAddress } from '../../utils/displayUtils';
 
 type DisplayAddressProps = {
   publicKey: PublicKey;
   isLinkable?: boolean;
 };
 
-// TODO: Get Identity type from react-sdk
-// TODO: Abstract this hardcoding away since we won't know types generally in the long run
-const displayName = (identity: any) => {
-  switch (identity.type) {
+const displayCustomIdentityName = (identity: Identity) => {
+  switch (identity.identityName) {
     case 'SNS': {
```

Comment:
Do we have an enum for these `identityName` values? I think it would make sense to add one in SDK, wdyt?